.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+
+> Provide a description of the purpose of this pull request, as well as its
+motivation and context. Is it a new feature? A bug fix? Does it address an existing issue?
+
+## How was the code tested?
+
+> Describe the conditions under which the code has been tested.
```

Comment:
this might be too general IMO. Perhaps explicitly asking for integration testing and/or unit testing makes it clear for the end user what it means. Thoughts?

Reply:
In this case, it is also good to know where the code was tested (e.g. LXD, AWS, VSphere, MAAS) and which OS (e.g. Ubuntu, Centos). It is general indeed, to provide as many details as possible. The integration and unit tests are prerequisites anyway.

Reply:
I can update the description to elicit more information from the contributor. I can ask what undercloud was used (microk8s, MAAS, AWS, etc.) and what OS the Juju unit is running. The testing substrate we will most likely use for the integration tests will be LXD. Canonical's Charmed Kubernetes team provides a GitHub action for bootstrapping a Juju controller: https://github.com/charmed-kubernetes/actions-operator.

Reply:
Updated the description of "How was the code tested" to include information that we are looking for in https://github.com/omnivector-solutions/slurmdbd-operator/pull/1/commits/94b3a7ab9fe06b5d24ca4ada1000acfa91d7ffdb. Let me know what you both think!

Reply:
@NucciTheBoss, I think it is great!